src-react/containers/path.js

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux'
 
+import "../styles/styles.scss";
```

Comment:
Updated for single quotes; I also add the `webpack` alias for styles.

Besides, let's fix the eslint issue in another PR?